src/Compilers/CSharp/Portable/Errors/CSDiagnosticInfo.cs

```diff
@@ -40,20 +40,10 @@ private CSDiagnosticInfo(bool isWarningAsError, ErrorCode code, object[] args, I
             _additionalLocations = additionalLocations;
         }
 
-        public override IReadOnlyList<Location> AdditionalLocations
-        {
-            get
-            {
-                return _additionalLocations;
-            }
-        }
+        public override IReadOnlyList<Location> AdditionalLocations => _additionalLocations;
 
-        internal new ErrorCode Code
-        {
-            get
-            {
-                return (ErrorCode)base.Code;
-            }
-        }
+        internal new ErrorCode Code => (ErrorCode)base.Code;
+
+        public static bool IsEmpty(DiagnosticInfo info) => (object)info == EmptyErrorInfo;
```

Comment:
It would be more clear to make this `internal`. The containing type is `internal`.